src/components/Dashboard/ThisWeek/NewsletterMenu/NewsletterMenu.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactElement } from 'react';
+import { GqlMockedProvider } from '../../../../../__tests__/util/graphqlMocking';
+import {
+  GetEmailNewsletterContactsQuery,
+  GetTaskAnalyticsQuery,
+} from '../NewsletterMenu.generated';
+import NewsletterMenu from './NewsletterMenu';
+
+export default {
+  title: 'Dashboard/ThisWeek/NewsletterMenu',
+};
+
+const accountListId = '111';
+
+export const Default = (): ReactElement => {
+  const mocks = {
+    GetTaskAnalytics: {
+      taskAnalytics: {
+        lastElectronicNewsletterCompletedAt: '2020-10-27T16:20:06Z',
+        lastPhysicalNewsletterCompletedAt: '2020-11-11T19:42:03Z',
```

Comment:
For some reason, ergonomock was always returning null, even though the type should be a ```string```. I think it may have something to do with it being a custom scalar, but not too sure. To fix it, I just had to mock the specific dates I wanted.